database/migrations/2021_04_21_183428_create_reactions_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateReactionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reactions', function (Blueprint $table) {
+            $table->bigInteger('post_id');
```

Comment:
Why did you used `->integer` in `comments` and `bigInteger` here?

Reply:
I just copied it from somewhere, i think.
I changed it to integer now